src/main/java/pl/byczazagroda/trackexpensesappbackend/repository/WalletRepository.java

```diff
@@ -0,0 +1,10 @@
+package pl.byczazagroda.trackexpensesappbackend.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+
+@Repository
+public interface WalletRepository extends JpaRepository<Wallet, Long> {
+
```

Comment:
Java doc f.e /** Repository for Wollet Entity */ . Please add another Javadoc for the rest public class and only if the are Superclass.  So that mean add to  the  WalletService  but for the WalletServiceImpl not.

Reply:
@Repository annotation isn't required here